lib/dispatcher.js

```diff
@@ -0,0 +1 @@
+module.exports = _.clone(Backbone.Events);
```

Comment:
Fair point, we should emphasise that in the README. Because I think it's fair to choose this.